pkg/apis/core/types_shoot.go

```diff
@@ -765,6 +765,10 @@ const (
 // KubeletConfig contains configuration settings for the kubelet.
 type KubeletConfig struct {
 	KubernetesConfig
+	// A quantity defines the maximum size of the container log file before it is rotated. For example: "5Mi" or "256Ki".
```

Comment:
To make `golint` happy:

```suggestion
	// ContainerLogMaxSize defines the maximum size of the container log file before it is rotated. For example: "5Mi" or "256Ki".
```

PS: I see that you used the doc string from kubernetes/kubernetes but I think we trying to start the doc strings with the field name.